### PR TITLE
* Update README with new maven version to use

### PR DESCRIPTION
**Pull request by ag-ramachandran**
#### Pull Request Description

_* Credentials provider is not supported in the new revision. Removed the reference_

---

**Breaking Changes:**
- None

**Features:**
- None

**Fixes:**
- Updates to README documentation

### CODE REVIEW

**Review by asaharn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This method is the easiest to follow, just add the following dependency to your
 <dependency>
     <groupId>com.microsoft.azure.kusto</groupId>
     <artifactId>kusto-ingest</artifactId>
-    <version>2.1.2</version>
+    <version>3.1.3</version>
```
Comment (yogilad) [inline README.md:26]: Should we put something here that does not require an update per version, e.g. 
``` xml
<version>[current sdk version]</version>, or 
<version>x.y.z</version>
```

Comment (ag-ramachandran) [reply README.md]: Good point @yogilad  , Added a badge to represent the version to maven central. Would this be fine

Comment (AsafMah) [reply README.md]: The badge is a great idea, but people who copy+paste will now not have the version field.

Maybe add `<version>[see badge below]</version>`

Comment (ag-ramachandran) [reply README.md]: Sure @AsafMah , good idea!

**Review by ohadbitt — APPROVED** [review]

**Review by AsafMah — APPROVED** [review]

**Pull request merged**